lightning/src/ln/channel.rs

```diff
@@ -1918,6 +1933,10 @@ impl<Signer: Sign> Channel<Signer> {
 			// channel.
 			return Err(ChannelError::Close("Received funding_created after we got the channel!".to_owned()));
 		}
+		if self.inbound_awaiting_accept {
+			// The Channel must manually accepted before the FundingCreated is received.
```

Comment:
nit: must *be manually accepted

I think we can get rid of this comment though

Reply:
I agree, I'll get rid of it.